internal/datastore/spanner/reader.go

```diff
@@ -185,25 +186,28 @@ func queryExecutor(txSource txFactory) common.ExecuteQueryFunc {
 			relCount := 0
 			defer span.SetAttributes(attribute.Int("count", relCount))
 
+			var resourceObjectType string
+			var resourceObjectID string
+			var relation string
```

Comment:
Does it mean anything that these declarations moved out of the loop?

Reply:
It means they are on the heap, which isn't good, but it also means we don't need to recalculate them every iteration, which is good